src/com/houarizegai/calculator/Calculator.java

```diff
@@ -96,20 +96,20 @@ private Calculator() {
         int[] x = {MARGIN_X, MARGIN_X + 90, 200, 290, 380};
         int[] y = {MARGIN_Y, MARGIN_Y + 100, MARGIN_Y + 180, MARGIN_Y + 260, MARGIN_Y + 340, MARGIN_Y + 420};
 
-        inText = new JTextField("0");
```

Comment:
which one do you think will be more good? the one with the default number? or simply blank?

Reply:
according to me blank will be nice , as if we type any value then it will show or else it will blank.

Reply:
I'm neutral to this. Hoping others will say this opinion. Thanks for the reply.

Reply:
thanks for suggestion